modules/ROOT/pages/as2/as2-connectorMule4.adoc

```diff
@@ -0,0 +1,187 @@
++
+[%header%autowidth.spread]
+|===
+|Key| Description
+|Self config - AS2 Partner Name| Sender’s AS2 system identifier. Used for AS2-From and AS2-To headers.
+|Self config - x509 alias| Sender’s Certificate and/or private key alias in the keystore. Required if using signing or encryption.
+|Self config - Email| Sender partner’s email.
+|Partner config - AS2 ID| Receiver’s AS2 system identifier. Used for AS2-From and AS2-To headers.
+|Partner config - x509 alias| Receiver’s Certificate and/or private key alias in the keystore. Required if using signing or encryption.
+|Partner config - Email| Receiver partner’s email.
+|Requester config - Subject| Free form text for MIME Subject header. Can include references to self / partner `simpleName` values along with the message `fileName`, for example, Sending file example.txt to partner via AS2.
+|Requester config - MDN MIC Signature Algorithm| The algorithm with which MIC is calculated for the MDN/receipt signature. `UNSIGNED` by default, that is, message is not signed.
+|Requester config - Request a receipt| Option to request a receipt from the partner. `SIGNED_OPTIONAL` by default.
+|Requester config - Request delivery URL| Asynchronous receipt delivery URL which points to the MDN listener end point.
+|Requester config - Encryption Algorithm| The encryption algorithm for encrypting the message. `UNENCRYPTED` by default.
+|Requester config - MIC Signature Algorithm| The algorithm with which MIC is calculated for the message signature. `UNSIGNED` by default, that is, message is not signed.
+|Requester config - Content transfer encoding| The content transfer encoding. `BINARY` by default.
+|Requester config - Compression type| The compression type. ZLIB is the only supported compression/decompression. `NONE` by default.
+|Keystore config - Keystore Path| The path of the keystore where certificates and keys are provided.
+|Keystore config - Keystore Password| The password for the keystore.
+|Keystore config - Private key Password| If private key requires a password.
+|===
++
+. Configure a connection to the partner AS2 endpoint by providing the `Partner Base URL`.
+. If you want to configure other connection details such as TLS, proxy, timeouts you can do that now.
+. Choose to send the file/message by either using `Send with sync mdn` or `Send with async mdn`:
+
+** The payload is the message content to be sent.
+
+** Filename is the file name of the message being sent.
+
+** Boolean to configure if the process should fail or if the receipt status is not processed.
+
+NOTE: If you choose to request an asynchronous receipt, you need to configure `MDN Listener`.
+
+=== Configure a MDN Listener
+
+Place the connector’s `MDN Listener` as a source in your flow as applicable for your use case and reference the global config defined here.
+
```

Comment:
Again per the style guide and previous edits, we need to stop presenting key-value pairs in two-column tables and use a description list (dlist) instead.

Omitting the table formatting, the dlist is formatted like this:

Self config - AS2 Partner Name:: Sender’s AS2 system identifier. Used for AS2-From and AS2-To headers.

It transforms like this:

Self config - AS2 Partner Name
Sender’s AS2 system identifier. Used for AS2-From and AS2-To headers.

GLOBAL reformat for table, or implement this copyedit for now.

Also, it is worthwhile to ask ourselves if we really need to provide our sophisticated audience with parameter definitions, especially when those parameters are intuitively named.